src/stories/SegmentedPicker/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import clsx from "clsx";
+import { SegmentedPickerProps } from "./type";
+
+export const SegmentedPicker: React.FC<SegmentedPickerProps> = ({
```

Comment:
please adjust to import FC directly

Reply:
change:export const SegmentedPicker: FC<SegmentedPickerProps<any>> = ({